src/components/Book.js

```diff
@@ -1,25 +1,56 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
 import { useDispatch } from 'react-redux';
-import { deleteBook } from '../redux/Books/booksSlice';
+import { removeBook } from '../redux/books/booksSlice';
+import Button from './Button';
 
-const Book = ({ book }) => {
-  const { title, author } = book;
+function Book({
+  id, title, author, category,
+}) {
   const dispatch = useDispatch();
   return (
-    <ul>
-      <li>{title}</li>
-      <li>{author}</li>
-      <li><button type="submit" onClick={() => dispatch(deleteBook(book))}>Delete</button></li>
-    </ul>
+    <div className="card-container">
+      <div className="book">
+        <p className="category">{category}</p>
+        <h3>{title}</h3>
+        <p className="author">{author}</p>
+        <div className="buttons">
+          <Button btnName="comments-btn" btnValue="comments" />
+          <div className="separate" />
+          <Button
+            type="button"
+            onClick={() => dispatch(removeBook(id))}
+            btnValue="Remove"
+            btnName="remove"
+          />
+          <div className="separate" />
+          <Button btnName="edit-btn" btnValue="Edit" />
+        </div>
+      </div>
+      <div className="progress-sec">
+        <div className="oval" />
+        <div className="percentages">
+          <h2>66%</h2>
+          <p>Completed</p>
```

Comment:
- [optional] Great job. Please kindly increase the font size of the text in blue sections so as to make them more visible and as well match the zeplin design well.